onnxruntime/test/providers/tensorrt/tensorrt_basic_test.cc

```diff
@@ -106,7 +106,7 @@ void RunWithOneSessionSingleThreadInference(std::string model_name, std::string
   RunOptions run_options;
   run_options.run_tag = so.session_logid;
   InferenceSession session_object{so, GetEnvironment()};
-  auto allocator_manager = session_object.GetAllocatorManager();
+  onnxruntime::AllocatorManager allocator_manager;
   auto cuda_provider = DefaultCudaExecutionProvider();
```

Comment:
AllocatorManager is just a container that is temporarily used to share allocators while calling RegisterAllocator. We don't need the AllocatorManager instance from the session, so removed GetAllocatorManager from that API as it was only used by these tests..